helm-eshell.el

```diff
@@ -71,6 +77,7 @@
    (nomark :initform t)
    (persistent-action :initform 'ignore)
    (nohighlight :initform t)
+   (fuzzy-match :initform (eval helm-eshell-fuzzy-match))
```

Comment:
Without `eval` fuzzy-matching is always enabled. It treats `helm-eshell-fuzzy-match` as `t` even it has value of `nil`.